### PR TITLE
Fix global-node-modules RPM build

### PR DESCRIPTION
**Pull request by kbgl**
Port `global-node-modules` changes from Debian to RPM build

### CODE REVIEW

**Review by kunal11191 — APPROVED** [review]
approving this since rhel and centos jobs are again stable on this branch. 
Please do wait for Ryans approval as well

**Comment by ryannowarm** [comment]
It might makes sense to add more detail to the commit message.  Something like: "upgrade global-node-modules to be inline with what we use in deb"

**Review by ryannowarm — CHANGES_REQUESTED** [review]
It might makes sense to add more detail to the commit message. Something like: "upgrade global-node-modules to be inline with what we use in deb"

**Review by ryannowarm — APPROVED** [review]

**Pull request merged**
